### DISCUSSION TITLE
WBook: 基于 Walrus 的图书馆

### DISCUSSION THREAD

**Discussion by 0xKaso** [creatorsand/.github#130] [Ideas]
**一句话 IDEA**
基于 Walrus 的图书永久存储索引平台，将人类的智慧结晶进行永久存储，并通过简易的方式方便所有人获取知识。

**我的角色**
产品经理，主要负责产品设计，需求分析。
后端与链工程师，主要负责数据的传输，索引，上链和展示。

**其他角色需求（不包含发起人）：**
无，已有 2 名小伙伴。

**个人简介**
全栈工程师，目前在一家基金做 DeFi 工程师，曾经上过以太坊周刊，SUI 和去中心化理念的坚定信仰者。

**产品是给谁用的？**
所有希望将自己的知识永久存储的用户，所有希望学习先进知识的个人。

**其他产品有什么痛点，用户为什么要选择你的产品？**
当前 z-lib 存在以下痛点：
- 数据存储中心化，存在丢失风险；
- 数据维护成本较高，不便用于公益平台；
- 数据可能因为中心化管制收到篡改，使得知识内容发生变化。

**我们的解决方案**

- 基于 Walrus 的去中心化存储，保证内容的永久存储与不可篡改性；
- 提供便捷的网页前端与后台筛选机制，确保数据的便捷上传与优质数据进行上链；

**Comment by v1xingyue** [comment]
ai打分

[a comment by an automated account is not shown]

**Comment by 0xKaso** [comment]
github: https://github.com/Honor-War/library-web
成员微信: kasoqian, cwxv123321
